mmtrack/datasets/mot_challenge_dataset.py

```diff
@@ -362,8 +363,17 @@ def evaluate(self,
                 results, resfile_path, metrics)
             print_log('Evaluate CLEAR MOT results.', logger=logger)
             distth = 1 - track_iou_thr
-
             accs = []
+            # support loading data from ceph
+            client = self.file_client.client
+            local_dir = tempfile.TemporaryDirectory()
+            # avoid using root paths
+            # use self.img_prefix.split('mmtracking')[1][1:] to del the
+            # beginning '/'
+            # when both paths begin with '/', use osp.join will
+            # ignore the first part
+            tmp_prefix = osp.join('mmtracking',
+                                  self.img_prefix.split('mmtracking/')[1])
```

Comment:
Besides I don't get why there is a string of `mmtracking` in `self.img_prefix`.
Does `self.img_prefix` equal to `data/MOT17/train`?

Reply:
self.img_prefix begin at /home/......
It is an absolute path.

Reply:
Can you locate since when, the `self.img_prefx` becomes an absolute path?
We pass `img_prefix='data/MOT17/train'` into dataset from the config.

Reply:
Okay, I will check it.